config/locales/en.yml

```diff
@@ -2237,6 +2244,10 @@ en:
     connect_wca_id: "Connect your WCA ID to your account!"
   payments:
     #context: localized names of payment providers
```

Comment:
This is meant to be attached to `payment_providers`.

Reply:
I second that. You have to move this comment down to where it originally belonged

Reply:
Fixed